test/shopping-list.test.js

```diff
@@ -0,0 +1,231 @@
+const chai = require('chai');
+const expect = chai.expect;
+const should = chai.should;
+const item = require('../js/shopping-list-item.js');
+const list = require('../js/shopping-list.js');
+
+let ShoppingListItem = item.ShoppingListItem;
+let ShoppingList = list.ShoppingList;
+
+describe('ShoppingListItem', () => {
+  const ShoppingItem = new ShoppingListItem();
+
+  it('should be a class', () => {
+    expect(ShoppingListItem).to.be.a('function');
+  });
+
+  it('should be an instance of ShoppingListItem class', () => {
+    expect(ShoppingItem instanceof ShoppingListItem).to.be.true;
+  });
+
+  describe('Item Name', () => {
+    let soup = new ShoppingListItem('name');
+
+    it('should have a name property', () => {
+      expect(soup.name).to.equal('name');
+    });
+
+    it('should be a string', () => {
+      expect(typeof soup.name === 'string').to.be.true;
+    });
+  });
+
+  describe('Item Description', () => {
+    let soup = new ShoppingListItem("Cambell's", 'Tomato Soup');
+
+    it('should have a description property', () => {
+      expect(soup.description).to.equal('Tomato Soup');
+    });
+
+    it('should be a string', () => {
+      expect(typeof soup.description === 'string').to.be.true;
+    });
+  });
+
+  describe('isDone property', () => {
+    let soup = new ShoppingListItem("Cambell's", 'Tomato Soup');
+
+    it('should be a property of ShoppingListItem class', () => {
+      expect(soup).to.have.own.property('isDone');
+    });
+
+    it('should be set to false by default', () => {
+      expect(soup.isDone).to.equal(false);
+    });
+
+    it('should be a boolean', () => {
+      expect(typeof soup.isDone === 'boolean').to.be.true;
+    });
+  });
+
+  it('should have a constructor method that accepts 2 arguments, name & description', () => {
```

Comment:
You should also have tests that determine what happens if you try to build one with no arguments.